src/foam/parsers/HTMLLexer.js

```diff
@@ -283,9 +256,11 @@ foam.CLASS({
           doctypePart: function(v) { return null; },
           cdata: function(v) { return null; },
           comment: function(v) { return null; },
-          attributes: function(v) { return null; },
-          attribute: function(v) { return null; },
-          value: function(v) { return null; }
+          attributes: function(v) { return v; },
+          attribute: function(v) {
+            return Attribute.create({ name: v[0], value: v[1] });
```

Comment:
Will `v[1]` be `undefined` or `null`? We should probably set `Attribute.value` to `null`.